File: HelpfulSense/app/src/main/java/com/andruid/magic/helpfulsense/eventbus/ActionEvent.java
package com.andruid.magic.helpfulsense.eventbus;

import com.andruid.magic.helpfulsense.model.Action;

public class ActionEvent {
    private final Action action;
    private final String command;

    public ActionEvent(Action action, String command) {
        this.action = action;
        this.command = command;
    }

    public Action getAction() {
        return action;
    }

    public String getCommand() {
        return command;
    }
}

File: HelpfulSense/app/src/main/java/com/andruid/magic/helpfulsense/eventbus/ActionEvent.kt
package com.andruid.magic.helpfulsense.eventbus

import com.andruid.magic.helpfulsense.database.entity.Action

/**
 * EventBus object for [Action] related events
 * @property action that is added/modified/removed
 * @property command event
 */
data class ActionEvent(
        val action: Action?,
        val command: String
)